web/app/login.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
```

Comment:
Consider to use Material components: 

1. inputs https://material.io/components/text-fields
2. buttons: https://material.io/components/buttons